nvpy/utils.py

```diff
@@ -48,11 +48,6 @@ def get_note_title_file(note, replace_filename_spaces):
         else:
             fn = unicode(fn)
 
-        if note_markdown(note):
-            fn += '.mkdn'
-        else:
-            fn += '.txt'
```

Comment:
Degraded. systemtags are updated by Simplenote. It is an array that contains "pinned" or "markdown". nvPY changes the file extension based on it.
The internal data model of nvpy takes a lot of work to grasp. I plan to improve it in v2.4.0 or later.